Casks/g/galasactl@0.38.0.rb

```diff
@@ -9,7 +9,7 @@
   url "https://github.com/galasa-dev/cli/releases/download/v#{version}/galasactl-darwin-#{arch}",
       verified: "github.com/galasa-dev/cli/releases/"
   name "Galasa Client"
-  desc "Client to interact with the Galasa ecosystem or local development environment. Version 0.38.0."
+  desc "Client to launch Galasa tests on a Galasa service or locally.. Version 0.38.0"
```

Comment:
Minor - looks like there should be one '.' after "locally" instead of two '..'

Reply:
fixed